pyvo/utils/protfeature.py

```diff
@@ -0,0 +1,52 @@
+from dataclasses import dataclass
```

Comment:
call the file `protofeature` as you did above in the link, or even spell it out fully as `prototype_feature`?

Reply:
Typo... Thanks!